zebra-chain/src/test.rs

```diff
@@ -0,0 +1,4 @@
+//!Chain functionality for fetching test vectors.
```

Comment:
Nitpick/Optional: Could we rename this file to `tests` for consistency?

Reply:
updated name to tests.rs